rest/rest_block.py

```diff
@@ -174,6 +176,14 @@ def _locked_poll(self, paging=False):
             try:
                 signals, paging = self._process_response(resp)
                 signals = self._discard_duplicate_posts(signals)
+
+                # add the include_query attribute if it is configured
+                if self.include_query is not None and signals is not None:
```

Comment:
Definitely. Kind of a weird edge case, since allow none is True...you'd have to explicitly set it to the empty string. Still, though. Making the change.